ocs_ci/deployment/baremetal.py

```
@@ -1563,23 +1561,48 @@ def clean_disk(
 
     Returns:
-        str or None: The detected disk path (e.g., "/dev/sdb") or None if not found.
+        str or None: The detected disk path (e.g., "/dev/sdb", "/dev/nvme*n0*") or None if not found.
 
     """
+    namespace = namespace or constants.DEFAULT_NAMESPACE
+
     logger.info(
-        f"Attempting to auto-detect a suitable /dev/sd* disk on worker node: {wnode.name}."
+        f"Attempting to auto-detect a suitable /dev/sd*, /dev/nvme*n0* "
+        f"disk on worker node: {wnode.name}."
```

Ruff (0.15.12)

[warning] Logging statement uses f-string

(G004)